app/views/users/show.html.haml

```diff
@@ -9,6 +9,28 @@
             = @user.nickname
       %p
         = markdown(@user.biography)
+      %h3 Social Media and Social Code
+      %ul.list-unstyled
+        %li
+          = @user.website_url
```

Comment:
Please make this links! Think about what the user should provide as data and how you want to display!
E.g. for twitter I would expect that I enter ``@bruckmayer`` and that it displays ``https://twitter.com/bruckmayer``. So you have to think about each of these services...
Just displaying it in a list does not add much value ...